src/azure-cli-core/HISTORY.rst

```diff
@@ -2,10 +2,9 @@
 
 Release History
 ===============
-
 2.0.39
 ++++++
-* Minor fixes
+* authentication: support authorization code flow for interactive login
```

Comment:
I would make this more clear:
```
* `login`: Improve flow when using interactive login. The browser will not be opened for you.
```

Question, does the auth code auto-populate the field?

Reply:
this is a new flow and device code is no longer involved